js/modal1.js

```diff
@@ -0,0 +1,107 @@
+import { fetchAvailableTimes } from "./apiService.js";
+
+console.log(document.querySelector('.booking__searchButton'));
+console.log(document.querySelector('.custom__date'));
+/*Ronjas modal script*/ 
+console.log("hello from modal 1"); //To see if it works
+
+const modal = document.querySelector('.booking-modal');
+const steps = document.querySelectorAll('.booking-modal__step')
+const step1 = document.querySelector('#step1');
+const step2 = document.querySelector('#step2');
+const step3 = document.querySelector('#step3');
+
+console.log(modal);
+
+export const handleBookButtonClick = ()=>{
+console.log('Hello from the book btn');
+  
+modal.style.display = 'block';
+step2.style.display = 'none';
+step3.style.display = 'none';
+};
+
+function showSteps(stepNumber){
+    modal.style.display='block';
+    steps.forEach((step, index) => {
+        step.style.display = index + 1 === stepNumber ? 'block' : 'none';
+        step.classList.toggle('booking-modal__step--active', index + 1 === stepNumber);
+    });
```

Comment:
Perhaps you should reconsider including this code; I'm not entirely sure of its purpose. 😊 I believe you could achieve toggling between display: none and display: block simply by targeting the class name assigned in the HTML.